MetaMask/metamask-extension
# fix: Inconsistent base currency for network token

## Explanation
Description

Problem: Inconsistent UX for native network token (ETH) vs other tokens (ie. WETH, APE, etc)

Expected Behavior: Settings toggle should be between “Crypto” vs “Fiat”. Also when on “Crypto”, the bigger number should always be the crypto token. When on “Fiat”, the bigger number should always be the fiat number.

Acceptance Criteria:



```
GIVEN I am on the general settings tab
THEN the Primary Currency settings should show either "Crypto" or "Fiat"
```
 

```
GIVEN I have "Fiat" toggled on
THEN the bigger currency number should always be the fiat currency
```

-->

### Before

[Uploading 235692395-c7f8f4da-a886-4e97-9454-d746499e3c34.mp4…
](https://user-images.githubusercontent.com/54408225/235692395-c7f8f4da-a886-4e97-9454-d746499e3c34.mp4)

### After

https://github.com/MetaMask/metamask-extension/assets/26223211/56bf562d-20af-4f7c-b4fd-ac1c47276fe7


## Manual Testing Steps

Import an ERC20 token
See how conversion is displayed in fiat
Click Send
Add recipient
Add amount -- see conversion is displayed in ETH instead of fiat
Click Next
See conversion is displayed in ETH instead of fiat
